### PR TITLE
Radio jammers jam the radio more

### PR DESCRIPTION
**Pull request by ghost**
Now radio jammers fully prevent outgoing communication. Radio jammers had an issue where if you spammed chat your message would eventually go through in a way that didn't jam the important words. Now, it's impossible to do that. This should also make the jammer not hilariously bad for it's price.
:cl: 
balance: Radio jammers fully halt outgoing communications now.
/:cl:

### CODE REVIEW

**Comment by Ty-the-Smonk** [comment]
why not just make it 100%?

**Comment by py01** [comment]
I wanted the rare letter to pop through for flavor

**Comment by Ty-the-Smonk** [comment]
Cant wait for the <1% chance for HELP MAINT to get not jammed

**Comment by ExcessiveUseOfCobblestone** [comment]
the alternative (and cheaper) option is to just disable the headset entirely. EMPs prob need to be a little less damning in headset muting to make this item a competitive offer.

**Comment by Ty-the-Smonk** [comment]
> the alternative (and cheaper) option is to just disable the headset entirely. EMPs prob need to be a little less damning in headset muting to make this item a competitive offer.

Yellow extracts are silent and instant emps

**Comment by Tlaltecuhtli** [comment]
make it jam guns

**Comment by py01** [comment]
as per suggested, radio jammers just stop all outgoing radio communication now

**Comment by Shadowflame909** [comment]
Space Jam

File: code/game/objects/items/devices/radio/radio.dm [hunk 1]
```diff
@@ -235,8 +235,7 @@
 	for(var/obj/item/jammer/jammer in GLOB.active_jammers)
 		var/turf/jammer_turf = get_turf(jammer)
 		if(position.z == jammer_turf.z && (get_dist(position, jammer_turf) < jammer.range))
-			message = Gibberish(message,100)
-			break
+			return
```
Comment (ShizCalev) [inline code/game/objects/items/devices/radio/radio.dm]: update the comment at 233

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/radio/radio.dm [hunk 2]
```diff
@@ -235,8 +235,7 @@
 	for(var/obj/item/jammer/jammer in GLOB.active_jammers)
 		var/turf/jammer_turf = get_turf(jammer)
 		if(position.z == jammer_turf.z && (get_dist(position, jammer_turf) < jammer.range))
```
Comment (ShizCalev) [inline code/game/objects/items/devices/radio/radio.dm]: should this be <=?

**Comment by YakumoChen** [comment]
Still isn't much better than the 2 TC EMP flashlight, which is silent and reusable (it recharges uses over time)

**Comment by ExcessiveUseOfCobblestone** [comment]
this is aoe and not needing a charge?

**Comment by AnturK** [comment]
Might be bit too strong now but i guess we can go other way again if that's the case.

**Pull request merged**